src/Page/Guide/View.elm

```diff
@@ -45,6 +40,26 @@ view language guide allGuides allStories =
 -- viewCategorySubHeader : Language -> Page.Guide.Data.Guide -> List (Html Msg)
 
 
+viewGuideHeader : Language -> Page.Guide.Data.Guide -> Html Msg
+viewGuideHeader language guide =
+    let
+        t : Key -> String
+        t =
+            translate language
+    in
+    div [ css [ outerBorderStyle, withMediaPrint (Just [ marginBottom (rem 0) ]) ] ]
+        [ div
+            [ css [ centerContent ] ]
+            [ primaryHeader [ css [ guideTitleStyle ] ] guide.title
+            , viewRow
+                ( markdownToHtml guide.summary
```

Comment:
The summart should be plain text - not markdown because we use it in meta as well.

Reply:
I'm not sure what you gain from rendering with `viewRow` - does it match the design yet?

Reply:
the viewRow is because in the design the story page header bit has three columns: summary, image and a link.

I could move the image from the main body up into the leader bit to make that more obvious but it was out of scope for this issue.

Reply:
(They may want to put rich content in the summary like italic or bold?)

Reply:
We won't allow that - because it's used in the meta description & also because that block should be the same for all guides in terms of design. In the CMS it is a plain string also, not a rich text editor.

Reply:
> the viewRow is because in the design the story page header bit has three columns: summary, image and a link.
> 
> I could move the image from the main body up into the leader bit to make that more obvious but it was out of scope for this issue.

This is the Guide page & are you proposing to fix the design with image and caption in another issue already raised?

Reply:
fixed